app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :likes_counter
+
+  def update_post_like_counter
+    post.update(likes_counter: post.likes.count)
+  end
+end
```

Comment:
OPTIONAL
- You can also consider using the helper method `after_save` to ensure that with every update related to this specific model `Like` then invoke the `update_post_like_counter` method immediately. 👍

Reply:
This change has been done well